src/convenience/webhook.ts

```diff
@@ -183,17 +187,27 @@ function webhookCallback<C extends Context = Context>(
     let initialized = false;
 
     return async (...args: any[]) => {
-        const { update, respond, unauthorized, end, handlerReturn, header } =
-            adapter(...args);
+        const {
+            update,
+            respond,
+            unauthorized,
+            badRequest,
+            end,
+            handlerReturn,
+            header,
+        } = adapter(...args);
         if (!initialized) {
             // Will dedupe concurrently incoming calls from several updates
             await bot.init();
             initialized = true;
         }
         if (header !== secretToken) {
             await unauthorized();
-            // TODO: investigate deno bug that happens when this console logging is removed
-            console.log(handlerReturn);
+            return handlerReturn;
+        }
+        const updateData = await update;
+        if (!updateData?.update_id) {
```

Comment:
Is this supposed to check for `undefined` or both `undefined` and `0`?
```suggestion
        if (updateData?.update_id === undefined) {
```

Reply:
actually we can check against >== 0, update should be positive number